migration/src/main/java/uk/ac/bbsrc/tgac/miso/migration/destination/MigrationAuthorizationManager.java

```diff
@@ -83,4 +83,9 @@ public void throwIfNotWritable(SecurableByProfile resource) throws IOException,
     return new ArrayList<>(unfiltered);
   }
 
+  @Override
+  public void throwIfNonAdminOrMatchingOwner(User owner) throws IOException, AuthorizationException {
+
+  }
```

Comment:
A method body can be blank whenever it has void return type and you don't want it to actually do anything. It's nicer to leave a comment line explaining why it's empty though, so I will do that